### DISCUSSION TITLE
Registration process in General - participants

### DISCUSSION THREAD

**Discussion by sql-aus-hh** [dataplat/DataSaturdays#157] [Ideas]
We will need a registration page for welcoming and asking participants for the relevant informations.

[a comment by an automated account is not shown]

**Comment by andrekamman** [comment]
Registration tools like Eventbrite can handle this I guess?

[a comment by an automated account is not shown]

**Comment by amtwo** [comment]
I'd have a strong preference for identifying integration opportunities with different systems, rather than building from scratch. 

One other key is that we endeavor to ensure that event organizers are the masters of their own data, rather than keeping attendee data in a central repository. Your event, your attendees, your data. 

Using Eventbrite, and even embedding the sign-up form into a web page is pretty straightforward. 

Perhaps there's a way to "syndicate" the event description & details from a landing page at `datasaturdays.com/nnnn` over to the appropriate Eventbrite event via an Eventbrite API. This would ensure that the EventBrite event has complete/up-to-date data without manually synching the event landing page to the EventBrite page.

Integrations like Eventbrite-->MailChimp are also doable (I use Zapier to do this for my User Group) to enable organizers to easily add event registrants to long-term group CRM. I wonder if there's a way to streamline that setup for organizers so they can set those integrations up on their own accounts.

[a comment by an automated account is not shown]

**Comment by suxstellino** [comment]
Eventbrite is definitely what we're looking for. It gives also good analytical views, which is great IMHO. Furthermore, the service returns also exports of subscribers' stats. I definitely think that sessionize+Eventbrite is one of the best combos.

[a comment by an automated account is not shown]

**Comment by way0utwest** [comment]
This is, IMHO, one of the hardest items to manage and build. While I would not be opposed to building this type of system, we ought to provide integration links for those that want choice.

At this point, Eventbrite and Meetup are both options, and I would vote for ensuring we provide integration points for those.

One side note, I do not know if these allow for agreement with a CoC or AHP, but that is one think I feel is important.

[a comment by an automated account is not shown]
